java/statestore/src/test/java/sleeper/statestore/dynamodb/DynamoDBFileReferenceStoreIT.java

```diff
@@ -366,7 +375,8 @@ void shouldFailToSplitFileWhichDoesNotExist() throws StateStoreException {
             assertThatThrownBy(() ->
                     store.splitFileReferences(List.of(
                             splitFileToChildPartitions(file, "L", "R"))))
-                    .isInstanceOf(StateStoreException.class);
+                    .isInstanceOf(SplitRequestsFailedException.class)
+                    .hasCauseInstanceOf(FileReferenceNotFoundException.class);
```

Comment:
We can make these exceptions the same between the different state stores. We can use SplitRequestsFailedException everywhere.